Coalition Games (#5834)
endless-sky/endless-sky: data/coalition/coalition missions.txt

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition Games"
	name "Coalition Athletes"
	description "Transport three athletes to <destination> by <date> where they will compete in the Coalition games."
	deadline
	passengers 3
	source
		near "Homeward" 3 12
		government "Coalition"
	destination "Far Home"
	to offer
		"coalition jobs" >= 35
		month == 7
	on offer
		require "Jump Drive"
		conversation
			`You're approached by a group consisting of a Saryd, Kimek, and Arach. The Saryd speaks, and the small box around his neck translates after he stops.`
			`	"The astonishing sojourner, with great joy we salute."`
			`	You greet the group, and the Kimek continues the conversation. "The season of joy and camaraderie, we find ourselves in. Upon us, the Coalition games are."`

> 			`	The Arach speaks last, asking, "To <destination> by <day>, we must go. Compete to our hardest, we will. Transport us, would you, if upon <payment>, you agreed on?"`

Reviewer: They mention the games in the previous line, so I figured the assumption of them competing in the games was pretty direct.

I wouldn't see any issue with stating it more directly, if someone better than me at the Coalition speech patterns wants to add it. ;)

					goto arachi
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen only kimek"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the Arachi game.)`
					goto arachi
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen only arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen saryd, kimek"
			choice
				`	(Take the train to the Arachi game.)`
					goto arachi
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen saryd, arachi"
			choice
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen kimek, arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen all"
			choice
				`	(Take the last train.)`
					goto wp
				
			
			label saryd
			apply
				set "coalition games: saryd"

> 			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley, where the Saryd events are being held. Within the valley lies a track, stretching from a long line of starting gates then splitting as it disappears into a forest which covers most of the valley. The track reappears close to another stadium located in the valley below. The airspace above the valley is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just those that want a clearer view .`

Reviewer: ```suggestion
			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley, where the Saryd events are being held. Within the valley lies a track, stretching from a long line of starting gates then splitting as it disappears into a forest which covers most of the valley. The track apparently loops back on itself before emerging from the forest, reappearing in front of a second stadium which is located against the wall of the valley just below this one. The airspace above the valley is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just those that want a clearer view .`
```

			`	Saryds begin striding out through the other side of the forest, crossing a finish line along the way. Several dozen cross the finish line, but from what you gather only the thirty highest scoring ones may proceed. The monitors switch over to a ground crew interviewing those who qualified. You notice Ulmodi on screen, cheerily speaking to the interviewers. He seems to have finished fourth.`
			`	With the event over, the crowd disperses under the supervision of Heliarch agents. The agents guide the audience to rooms with many large elevators. You board one of the elevators, which begins descending down into the valley. During the descent, a Saryd Heliarch agent approaches you and begins speaking through her translation device. "Enjoyed the event have you, human?"`
			choice
				`	"It was a very entertaining event. I can see why the crowd was excited the whole time."`

> 				`	"I think the event would have been more interesting had I known what exactly was going on."`

Author: I tried my hand at some short responses, not anything major as I don't want to make the already long walls of text longer with the choices.

> 			`	As the elevator finishes it's descent and the doors begin to open, the Heliarch catches your attention once more. "Hope that you enjoy this next event, I do very much, though the best at it in my time I was not," she says as she begins guiding the audience along a path leading to the next stadium. In the center, you see a rectangular court of some sort with small mounds dotted around the field and six large holes arranged evenly around the sides. Tall nets stretching to the ceiling separate the field and the seating area in a way you find to be slightly worrying.`

Reviewer: I'll clear that up a bit :)

> 			`	As the elevator finishes it's descent and the doors begin to open, the Heliarch catches your attention once more. "Hope that you enjoy this next event, I do very much, though the best at it in my time I was not," she says as she begins guiding the audience along a path leading to the next stadium. In the center, you see a rectangular court of some sort with small mounds dotted around the field and six large holes arranged evenly around the sides. Tall nets stretching to the ceiling separate the field and the seating area in a way you find to be slightly worrying.`

Reviewer: ```suggestion
			`	As the elevator finishes its descent and the doors begin to open, the Heliarch catches your attention once more. "Hope that you enjoy this next event, I do very much, though the best at it in my time I was not," she says as she begins guiding the audience along a path leading to the other stadium in the valley. Fortunately, the path is much shorter than the track out and through the forest, so it isn't long before the escorted crowd arrives and begins filling the seats inside. In the center of this stadium, you see a rectangular court of some sort with small mounds dotted around the field and six large holes arranged evenly around the sides. Tall nets stretching to the ceiling separate the field and the seating area in a way you find to be slightly worrying.`
```

			`	Half an hour passes before the thirty Saryds, holding what looks like fancy polo mallets, enter the grounds and are divided into six teams of different colors. The members of each team are introduced to the crowd and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`
			`	The game starts. It's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience, but the nets are sturdy, and prevent any accidents.`
			`	Eventually, a sphere finds its way into a hole, which lights up with the color of the team who scored: red. Much of the crowd jumps up and screams. You're not sure if everyone is a fan of the red team or if they're cheering simply because a point was scored. The sphere is brought back up automatically via a tube close to the hole and it's back in the game.`

> 			`	The game continues, and each team accumulates a few points. The blue team, leading, scores a fifth point, and the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The Kimek with the translator explains what is happening.`

Reviewer: As the game continues each team scores a few times, but the blue team pulls into the lead. When they score their fifth point, the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The helpful Kimek with the translator excitedly explains to you what is happening.`

> 			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, going from the team that scored the least all the way to the blue team at the end. Between the two events, Ulmodi is placed third. He and the rest of the top twenty contestants celebrate by running laps around the arena.`

Reviewer: "The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, starting with the yellow team, the lowest scoring, and progressing up to the blue team at the end. After this event, Ulmodi has risen to being placed third, and you notice that the Saryd who caught your eye in the forest earlier is in first place. After the individual scores are announced, the top twenty contestants celebrate by running laps around the arena.`

> 			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena. The crowd begins to file out. You get back to where the train dropped you off and see that another one has arrived on the opposite side. A Heliarch agent explains that it is headed to the final arena where the cooperative game will take place.`

Reviewer: After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena, at which point the crowd begins to file out, once more under Heliarch direction. You arrive back at where the train dropped you off and see that another one has arrived on the opposite side. The Heliarch agent directing your group explains that it is headed to the final arena where the cooperative game will take place, but that you are also free to travel back to the other arenas and view the rest of the other games.`

> 			`	Once the game starts, as near as you can tell, it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`

Reviewer: I'm not sure what is being asked here? All 30 Saryds enter at once.

> 			`	Once the game starts, as near as you can tell, it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`

Reviewer 2: Sorry, I must've started this review before that was edited. I'll edit the review then. Apologies.

> 			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena, at which point the crowd begins to file out, once more under Heliarch direction. You arrive back at where the train dropped you off and see that another one has arrived on the opposite side. The Heliarch agent directing your group explains that it is headed to the final arena where the cooperative game will take place, but that you are also free to travel back to the other arenas and view the rest of the other games.`

Reviewer: Train from initial point to each individual game and back, and from each game to combined and back.

Train from each game to each other would be logical, but not strictly necessary, and I'm not sure what original intent behind stated structure was.

Maybe they simply don't want to have to manage three more trains and six more entrance/exit areas? Or direct routes between each area aren't feasible logistical?

			`	You ask a nearby Heliarch agent where Pairri will be competing, and they check a computer before telling you that of the twenty floors, she's on the sixteenth and pointing you towards the nearest elevator. After an elevator ride as crowded as the train in, you rush to find vacant a seat, because, though a quick glance shows there are likely enough seats on this level alone for over a hundred thousand spectators, you don't want to risk having to fight for a space given how quickly the seating area is filling up with fans all eagerly waiting for the event to begin. As you settle in, you spot many groups of Arachi and several lone Saryds, but compared to the thousands upon thousands of Kimek crowded around them, from a distance they look nearly as out of place as you feel yourself. `

> 			`	After a few minutes, bridges extend from just under the seating area of the next level above you to the circular platform in the middle of the room, and hundreds of Kimek contestants walk out upside-down, clinging to underside of the bridges and platform, as Heliarch agents divide them into two teams, numbering several hundred contestants each. The Heliarchs leave, and, after a long whistle is blown, the two teams of hanging Kimek charge at one another with surprising speed and agility, despite their swarming numbers and inverted orientation.`

Reviewer: I thought about adding something to expand on that, but thought it was best to leave a mystery to the player, as it didn't feel significant enough to the story or the outcome of the game to have a nearby fan explain it.

			`	This process repeats several more times until finally, those that remain - less than a tenth of those that entered - are gathered in the center. As they leave the ring, without an announcement of any rankings or score, you become unsure if the game has ended.`
			`	You feel something tap the top of your head and turn to find it is an Arach sitting behind you. "Confused, the human visitor is?" their little box translates.`
			choice
				`	"Yes. I have no idea what is going on."`
				`	"I think I somewhat understand, but an explanation would be helpful."`
			`	They breakdown the competition for you during what you learn is a brief break. "Numerous, the Kimek are, so many, many athletes competing, they have," they say. "Over a thousand, at first enter each floor. Redone, the teams are, once in half, that number is cut. The true game begins, when one hundred remain."`

> 			`	As if on cue, the remaining contestants reenter the ring, this time wearing what looks like some kind of goggles, perhaps as protective gear. This time, instead of being split into two teams to fight, they are divided into pairs, and each pair moves to one of the small rings painted on the platform.`

Reviewer: ```suggestion
			`	Just then, the remaining contestants reenter the ring, almost as if on queue, now wearing what looks like some kind of goggles, perhaps as protective gear. This time, instead of being split into two teams to fight, they are divided into pairs, and each pair moves to one of the small rings painted on the platform.`
```

			`	Pairri draws your attention back to the line, where judges have the twenty teams line up. A chariot carrying the other two members of each team is attached to the Saryds, who seem to be working themselves up, hitting their hoofs repeatedly on the ground. Meanwhile, the Kimek and Arachi are securing large bags and pieces of fabric to their chariots.`
			`	"In many ways we differ, when it comes to physical form," Pairri says. "So in the cooperative match, different tasks, each member of a team is assigned. For the sake of fairness, it is so."`
			choice
				`	"What are they carrying in those bags?"`
				`	"So is pulling those chariots the 'task' of the Saryds?"`
			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping toward the wall. The Kimek and Arachi of each chariot do their best to stay on, as their Saryd teammates press on with little regard for the chariot's stability, focused on reaching the wall as fast as possible.`

> 			`	It's a very tight race, but the Saryds begin breaking once they're getting closer, out of both exhaustion, and a need to avoid crashing into the wall. Once they arrive at the base, they immediately lay down to rest as the Arachi of each team unstrap the chariots, and the Kimek begin climbing the wall, carrying their small bags.`

Author: I think one actually crashing into the wall would be a bit too much for an otherwise really tame mission.

> 			`	It's a very tight race, but the Saryds begin breaking once they're getting closer, out of both exhaustion, and a need to avoid crashing into the wall. Once they arrive at the base, they immediately lay down to rest as the Arachi of each team unstrap the chariots, and the Kimek begin climbing the wall, carrying their small bags.`

Reviewer: Easy enough to nix or even adjust to just a less serious collision. Let me know which you prefer. :)

Really just added it as a tie back to the one swiping the tree since the need for braking was already emphasized there in the first version.

> 			`	It's a very tight race, but the Saryds begin breaking once they're getting closer, out of both exhaustion, and a need to avoid crashing into the wall. Once they arrive at the base, they immediately lay down to rest as the Arachi of each team unstrap the chariots, and the Kimek begin climbing the wall, carrying their small bags.`

Author: I'd just go without it, this portion is a race in an open field, and the athletes know how the games go, so this should be one of the easiest portions to train for, as it is only a sprint (granted a very taxing one) where they need to slow down at a point. Given only 20 Saryds make it to this portion, that'd also reduce the likelihood of any of them messing this part up, given they were the best of their own game.

> 			`	It's a very tight race, but the Saryds begin breaking once they're getting closer, out of both exhaustion, and a need to avoid crashing into the wall. Once they arrive at the base, they immediately lay down to rest as the Arachi of each team unstrap the chariots, and the Kimek begin climbing the wall, carrying their small bags.`

Reviewer: Done.

			`	Before long, they have all reached the top, opening their bags to grab a multitude of paddles, a pole that looks like a combination between a shoven and a pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must."`

> 			`	Back at the bottom, the Arachi are done helping their Saryd teammates out of the chariot straps, and prepare for the climb themselves, as they 'wear' the fabric pieces over their bodies, on which the Saryds lay, to be carried up by their teammates. You have your monitor focus on Pairri's friends again, and see that after ensuring Ulmodi is safely strapped on his back, Tumub begins his climb. Despite the stress of carrying someone much larger than him, his movements remain precise, never having more than one leg not touching the wall at a time, and carefully choosing the best spots on the wall to anchor himself on.`

Reviewer: ```suggestion
			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems the Arachi carry their larger teammates! You have your monitor focus on Pairri's friends again, and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he carefully chooses the best spots to anchor himself and support the weight of his Saryd friend.`
```

> 			`	Once they're at the top, Ulmodi gets up and removes the fabric from Tumub, now being his turn to carry his Arach teammate toward the now nearly complete raft. He lays the Tumub beside the raft and helps the Kimek properly secure the ropes. Once they're done, they help Tumub onto it, and start pushing the raft to the river. Ulmodi gets on it shortly before it touches the water, and the Kimek waits until the last second.`

Reviewer: ```suggestion
			`	Once Tumub reaches the top, Ulmodi gets dislodges himself and removes the harness from Tumub, before then lifting his Arach teammate into his arms so that he may climb onto the Saryd's back this time, as Ulmodie races to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft and both their Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where Ulmodi now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`
```

> 			`	The Kimek from their group has picked up three of the paddles and is working to fight the river's flow, without much success. Ulmodi grabs the pole tool from between two of the logs, and uses the pick edges to grab onto some rocks sticking up, pulling and tilting the boat back on course. With the shovel portion, he thrusts at the rocks to push the boat against the current. About halfway across, Tumub has caught his breath and goes to help the Kimek with the paddles, grabbing the four remaining ones. With his assistance, the paddles now do make a bit of a difference against the current, and while not enough to completely counter it, the raft is slowed enough that Ulmodi can hook the rocks more easily.`

Reviewer: ```suggestion
			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. Meanwhile Ulmodi grabs the pole tool from between two of the logs and uses the pick edges to grab onto some of rocks jutting from the river, in order to stabilize the raft and help pull them across the swift current, while alternately thrusting at the rocks with the shovel portion in order to keep their small raft from dashing against them. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft both becomes more stable and begins to make it's way across the river even when Ulmodi has no convenient rock to pull against.`
```

> 			`	You switch back to a more broad view of the race, and see that not everyone is having the same success crossing: two rafts, unable to fight the stream, drift dangerously far down the river, getting close to the waterfall. The Heliarch boats speed toward them, rescuing the competitors before anyone is hurt. Some team have the raft fall apart, as the ropes weren't tightened enough around the logs, and the Saryd of one team cannot hit a rock in time, and her team's raft ends up crashing onto it. More Heliarch boats take care of the adrift competitors, bringing them back to shore.`

Reviewer: ```suggestion
			`	You switch back to a more broad view of the race, and see that not everyone is having the same success crossing. Two rafts, unable to fight the stream, drift dangerously far down the river and close to the waterfall, while others have been shattered or torn into by the rocks when the Saryd on board cannot manage to push away from the rocks in time.  One team's raft even simply falls apart without hitting a single rock; it seems the ropes weren't properly fastened to the logs. The Heliarch safety boats on hand speed towards those in distress, plucking the competitors from the river's currents before anyone can be seriously hurt.`
```

> 			`	In total, twelve teams make it across the river, with Ulmodi, Tumub, and their Kimek parter winning second place by a hair. The competitors celebrate with their team and later gather to commend their opponents while the crowd around you cheers in all three Coalition languages. Everyone grows quiet, however, as three Heliarch Punishers land right in front of the athletes. Three Heliarchs, wearing circlets on their heads, come down the ships to greet the contestants.`

Reviewer: ```suggestion
			`	As the competitors begin crossing the final line on the far bank of the river, they celebrate with their team and then gather to commend their opponents while the crowd around you cheers in all three Coalition languages each time a new team finishes. In total, twelve of the twenty teams make it across the river, with Ulmodi, Tumub, and their Kimek partner achieving one of the loudest swells of applause and cheering from the crowd as they win second place "by half a Kimek," as Pairri phrases it. Everyone grows quiet, however, when three Heliarch Punishers land right in front of the athletes. A trio of Heliarchs representing each species and wearing circlets on their heads then come down from each of the ships to greet the contestants.`
```

> 			`	"Deemed worthy, some of them have been," Pairri says. "Name those they have chosen, the Heliarchs will." As the three Heliarch captains speak, they commend the teams that have made it there, noting that their accomplishments are proof of what they can achieve when cooperating. Once they're done, the monitor switches to a live view of one of the athletes, a Kimek. A Saryd is next, then another Kimek, followed by an Arach. Tumub's and Ulmodi's faces are shown next, at which point Pairri repeatedly taps your arm with one of her legs.`

Reviewer: ```suggestion
			`	"Deemed worthy, some of them have been," Pairri says. "Name those they have chosen, the Heliarchs will." As the Heliarch captains from each ship speak, they commend the teams that have made it there, noting that their accomplishments are proof of what everyone in the Coalition can achieve when cooperating as one. Once they're done, the monitor switches to a live view of one of the athletes, a Kimek. A Saryd is next, then another Kimek, followed by an Arach. Tumub's and Ulmodi's faces are shown next, at which point Pairri repeatedly taps your arm with one of her legs.`
```

			`	"Selected, my friends have been!" She cheers. "Made it, they have!"`

> 			`	Most of the audience shares the excitement, as if roaring in a chorus to congratulate the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony, announcing the end of this year's Coalition games, and invite six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs, and will go to the Ring of Friendship next to get started on their new careers.`

Reviewer: ```suggestion
			`	Most of the audience shares her excitement, and their unified cheers are a roaring chorus of congratulations for the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony and announce the end of this year's Coalition games, then invite two each of the six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs, and will go to the Ring of Friendship to begin training their new positions of honor.`
```

> 			`	The ring platforms head back to the hangars, and you board a train with Pairri to head back to one of the previous arenas, and from there to the entrance to the games. During the train rides she used some device to call friends and family members, who congratulate her on getting so far. She can't seem to contact Ulmodi or Tumub, though.`

Reviewer: ```suggestion
			`	The ring platforms head back to the hangars, then you and Pairri board a train head back to one of the previous arenas and from there to the entrance to the games. During the train rides she uses some device to call friends and family members, who congratulate her on getting so far, but even after trying several times, she can't seem to contact Ulmodi or Tumub.`
```

> 			`	Once Tumub reaches the top, Ulmodi gets dislodges himself and removes the harness from Tumub, before then lifting his Arach teammate into his arms so that he may climb onto the Saryd's back this time, as Ulmodie races to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft and both their Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where Ulmodi now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`

Reviewer: Dropped word. Should be "both help their Kimek teammate"

> 			`	Once Tumub reaches the top, Ulmodi gets dislodges himself and removes the harness from Tumub, before then lifting his Arach teammate into his arms so that he may climb onto the Saryd's back this time, as Ulmodie races to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft and both their Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where Ulmodi now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`

Author: At this point Tumub is still exhausted, so only Ulmodi would be helping the Kimek.
I'll change some bits to have it be that way, might mess up some grammar or wording in that so tell me if that happens.

			`	"Busy with the Heliarchs they must be." She guesses. "That there with them I am not, a shame it is."`
			choice
				`	"So are they Heliarchs now?"`
				`	"You almost won this time, so I'm sure you'll make it next time."`
					goto nexttime
			`	"Heliarchs, yes, but properly trained ones, not quite," She explains. "Still instructed in their new functions, they must be. Similar to the standard process, it is. Only, for those selected during the games, much faster said process is."`
				goto ending
			label nexttime
			`	"Thank you Captain. Even if next year watching my matches you are not, my best, I promise I will do."`
			label ending

> 			`	When you're nearing the docks, a dozen Kimek suddenly come running towards you two, and Pairri tells you it's her family. She tries to introduce you to all of them, but they seem much more interested in telling her how proud they were to see her compete than they are in meeting you. After they calm down a bit, you say goodbye to Pairri, wish her good luck for next year's games, and head back to your ship.`

Reviewer: ```suggestion
			`	When you're nearing the docks, a dozen Kimek suddenly come rushing towards two of you, and Pairri tells you it's her family. She tries to introduce you to all of them, but most of them seem much more interested in telling her how proud they were to see her compete than they are in meeting you, despite your alien appearance. Only one of the younger Kimek in the family group pays you particular mind over Pairri, but in all the hustle they still don't have a chance to speak to you with her translation box. After her family calms down a bit and begin to whisk Pairri away, you say goodbye, wishing her good luck for next year's games, and then head back to your ship.`
```

> 			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. Meanwhile Ulmodi grabs the pole tool from between two of the logs and uses the pick edges to grab onto some of rocks jutting from the river, in order to stabilize the raft and help pull them across the swift current, while alternately thrusting at the rocks with the shovel portion in order to keep their small raft from dashing against them. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft both becomes more stable and begins to make it's way across the river even when Ulmodi has no convenient rock to pull against.`

Reviewer 2: ```suggestion
			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. At the same time, Ulmodi grabs the pole tool from between two of the logs and uses the pick edges to grab onto some of rocks jutting from the river, while alternately thrusting at other rocks with the shovel portion. This has the effect of stabilizing the raft, as well as helping to pull them across the swift current and keeping their small raft from dashing against the rocks. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft begins to make its way across the river even when Ulmodi has no convenient rock to pull against.`
```

Author: Since you suggested two different changes to this line, should I pick whichever? Is one preferrable to the other?

Reviewer 2: Oh, my bad, that was unintentional. I'll delete one of them then.

> 			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. Meanwhile Ulmodi grabs the pole tool from between two of the logs and uses the pick edges to grab onto some of rocks jutting from the river, in order to stabilize the raft and help pull them across the swift current, while alternately thrusting at the rocks with the shovel portion in order to keep their small raft from dashing against them. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft both becomes more stable and begins to make it's way across the river even when Ulmodi has no convenient rock to pull against.`

Reviewer 2: ```suggestion
			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. At the same time, Ulmodi grabs the pole tool from between two of the logs. He then uses the pick edges to grab onto some of rocks jutting from the river, while alternately thrusting at other rocks with the shovel portion in order to keep their small raft from dashing against them. This stabilizes the raft and helps pull them across the swift current. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft begins to make its way across the river even when Ulmodi has no convenient rock to pull against.`
```

> 			`	As the competitors begin crossing the final line on the far bank of the river, they celebrate with their team and then gather to commend their opponents while the crowd around you cheers in all three Coalition languages each time a new team finishes. In total, twelve of the twenty teams make it across the river, with Ulmodi, Tumub, and their Kimek partner achieving one of the loudest swells of applause and cheering from the crowd as they win second place "by half a Kimek," as Pairri phrases it. Everyone grows quiet, however, when three Heliarch Punishers land right in front of the athletes. A trio of Heliarchs representing each species and wearing circlets on their heads then come down from each of the ships to greet the contestants.`

Reviewer 2: ```suggestion
			`	As the competitors begin crossing the final line on the far bank of the river, they celebrate with their team and then gather to commend their opponents. The crowd around you cheers in all three Coalition languages each time a new team finishes; in total, twelve of the twenty teams make it across the river. Ulmodi, Tumub, and their Kimek partner achieve one of the loudest swells of applause and cheering from the crowd as they win second place "by half a Kimek," as Pairri phrases it. Everyone grows quiet, however, when three Heliarch Punishers land right in front of the athletes. A trio of Heliarchs representing each species and wearing circlets on their heads then come down from each of the ships to greet the contestants.`
```

> 			`	The ring platforms head back to the hangars, then you and Pairri board a train head back to one of the previous arenas and from there to the entrance to the games. During the train rides she uses some device to call friends and family members, who congratulate her on getting so far, but even after trying several times, she can't seem to contact Ulmodi or Tumub.`

Reviewer 2: ```suggestion
			`	The ring platforms head back to the hangars. You and Pairri board a train that takes you back to one of the previous arenas, and from there you take another train back to the entrance to the games. During the train rides she uses some device to call friends and family members, who congratulate her on getting so far, but even after trying several times, she can't seem to contact Ulmodi or Tumub.`
```

mission "Coalition Games"
	name "Coalition Athletes"
	description "Transport three athletes to <destination> by <date> where they will compete in the Coalition games."
	deadline
	passengers 3
	source
		near "Homeward" 3 12
		government "Coalition"
	destination "Far Home"
	to offer
		"coalition jobs" >= 35
		month == 7
	on offer
		require "Jump Drive"
		conversation
			`You're approached by a group consisting of a Saryd, Kimek, and Arach. The Saryd speaks, and the small box around his neck translates after he stops.`
			`	"The astonishing sojourner, with great joy we salute."`
			`	You greet the group, and the Kimek continues the conversation. "The season of joy and camaraderie, we find ourselves in. Upon us, the Coalition games are. To <destination> by <day>, we must go, for competing in the games, we are. Transport us, would you? Pay you <payment>, we would."`
			choice
				`	"That sounds good. I'll bring you there."`
					goto end
				`	"Coalition games?"`
				`	"Sorry, I'm not looking for any transport jobs right now."`
					decline
			`	"To determine the athlete with the most merit, a competition it is," the Saryd says.`

> 			`	"A candidate for the rank of Heliarch, one becomes, if among the victors, othey are. Trained hard, we have, and to prove our worth in the games, we wish," the Arach adds, and they once again look at you, expecting an answer to their original question.`

Reviewer 3: ```suggestion
			`	"A candidate for the rank of Heliarch, one becomes, if among the victors, they are. Trained hard, we have, and to prove our worth in the games, we wish," the Arach adds, and they once again look at you, expecting an answer to their original question.`
```

			choice
				`	"Sounds like you're all really excited about this. Alright, I'll bring you there."`
				`	"Well, I wish you luck then, but sadly I'm not looking for any transport jobs right now."`
					decline
			label end
			`	"Most grateful, we are," the Saryd says.`
			`	They introduce themselves as you help bring their luggage into your ship; the Saryd is called Ulmodi, the Kimek is Pairri, and the Arach is Tumub. They settle in their bunks to rest for the upcoming event, and you plot out a route to <destination>.`
				accept
	on complete
		"coalition jobs" += 3
		payment 300000
		conversation

> 			`While Ulmodi begins unloading the athletes' luggage and Tumub counts out your payment, Pairri approaches you. She stretches her legs trying to raise herself up your height, and you humor her by crouching down. In response, she offers you a small coin that you don't recognize, then says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."  The coin has a symbol similar to the old Olympic Games on Earth, except with three interlocked rings instead of five.`

Reviewer 3: ```suggestion
			`While Ulmodi begins unloading the athletes' luggage and Tumub counts out your payment, Pairri approaches you. She stretches her legs trying to raise herself up to your height, and you humor her by crouching down. In response, she offers you a small coin that you don't recognize, then says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."  The coin has a symbol similar to the old Olympic Games on Earth, except with three interlocked rings instead of five.`
```

			`	"Separate competitions, each species has," Tumub says. "To value each one's history, it is so."`
			`	"A grand, cooperative match, at the end of it all takes place," Ulmodi further details. "Teams consisting of all three species, formed they are. A joyous, unmatched experience, the last spectacle is."`
			`	Since they need to go their separate ways and prepare for the games, they don't have time to explain each of the individual sports, but assure you that watching them will be enjoyable even if you don't understand the rules. Nearby, you find a Heliarch agent who quickly points you to the line for ticket admission, which you're shocked to see is an individual queue of thousands of people from all three Coalition species. Despite the length, the Heliarchs managing admissions are incredibly efficient and organized; you spend barely ten minutes waiting to get in.`
			`	After turning in your ticket at the main gate, another Heliarch shows you where you should go for each competition. Three massive transports, similar in design to old-fashioned bullet trains, wait to take spectators to a different arena for each set of games.`

> 			`	They also explain that, from there, you'll take another train to go to the "grand spectacle" that serves as a central finale for the Coalition Games once the individual matches are over. You thank them for the explanation and decide on which train to take.`

Reviewer 3: ```suggestion
			`	They also explain that, from there, you'll take another train to go to the "grand spectacle" that serves as a central finale for the Coalition Games once the individual matches are over. You thank them for the explanation and decide which train to take.`
```

			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the Kimek game.)`
					goto kimek

> 				`	(Take the train to the Arachi game.)`

Reviewer 3: ```suggestion
				`	(Take the train to the Arach game.)`
```

No 'i' since it's the adjective. You also use the plural form in the conditions while 'saryd' is kept singular, but that's less important.

					goto arachi
			
			
			label choice
			branch "seen all"
				has "coalition games: saryd"
				has "coalition games: kimek"
				has "coalition games: arachi"
			
			branch "seen saryd, kimek"
				has "coalition games: saryd"
				has "coalition games: kimek"
			
			branch "seen saryd, arachi"
				has "coalition games: saryd"
				has "coalition games: arachi"
			
			branch "seen only saryd"
				has "coalition games: saryd"
			
			branch "seen kimek, arachi"
				has "coalition games: kimek"
				has "coalition games: arachi"
			
			branch "seen only kimek"
				has "coalition games: kimek"
			
			branch "seen only arachi"
				has "coalition games: arachi"
			
			label "seen only saryd"
			choice
				`	(Take the train to the Kimek game.)`
					goto kimek

> 				`	(Take the train to the Arachi game.)`

Reviewer 3: ```suggestion
				`	(Take the train to the Arach game.)`
```

					goto arachi
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen only kimek"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd

> 				`	(Take the train to the Arachi game.)`

Reviewer 3: ```suggestion
				`	(Take the train to the Arach game.)`
```

					goto arachi
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen only arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen saryd, kimek"
			choice

> 				`	(Take the train to the Arachi game.)`

Reviewer 3: ```suggestion
				`	(Take the train to the Arach game.)`
```

					goto arachi
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen saryd, arachi"
			choice
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen kimek, arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen all"
			choice
				`	(Take the last train.)`
					goto wp
				
			
			label saryd
			apply
				set "coalition games: saryd"

> 			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley where the Saryd events are being held. Within the valley lies a track, which stretches from a long line of starting gates then splits as it disappears into a forest covering most of the valley. The track apparently loops back on itself before emerging from the forest, reappearing at the other side. The finish line is in front of a second stadium located against the wall of the valley, just below the first stadium. The airspace above is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just want a clearer view .`

Reviewer 3: ```suggestion
			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley where the Saryd events are being held. Within the valley lies a track, which stretches from a long line of starting gates then splits as it disappears into a forest covering most of the valley. The track apparently loops back on itself before emerging from the forest, reappearing at the other side. The finish line is in front of a second stadium located against the wall of the valley, just below the first stadium. The airspace above is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just want a clearer view.`
```

			`	Just then, the remaining contestants reenter the ring almost as if on queue. This time, instead of being split into two teams to fight, they are divided into pairs, and each pair moves to one of the small rings painted on the platform.`
			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the bottom platform display each fighter and their changing scores as contestants rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

> 			`	After many of these rounds and rotational swap outs, you begin to think the the game has ended, as the contestants are milling around instead of continuing to pair off. Then, Pairri and one other, much larger Kimek move to a circle in the center of the field. "Tied for the highest score, they are," the helpful Arach behind you points out. "The floor victor, the winner of this final match will be."`

Reviewer 3: ```suggestion
			`	After many of these rounds and rotational swap outs, you begin to think the game has ended, as the contestants are milling around instead of continuing to pair off. Then, Pairri and one other, much larger Kimek move to a circle in the center of the field. "Tied for the highest score, they are," the helpful Arach behind you points out. "The floor victor, the winner of this final match will be."`
```

			`	You approach a cluster of nine colossal trees which stand out from the rest. One in the center is taller than all the others, and the other eight are varying heights arranged regularly around it. Smaller rafts are also approaching the site from elsewhere, each with a different circular symbol on it. The Heliarch agents on your own raft are busy preparing to land, so you ask a Saryd seated next to you what they mean after noticing she's carrying a translation box.`
			`	"The multiple Arach houses, the symbols represent," she says. "To be patrons of athletes, a custom of members of the great houses it is."`

> 			`	The rafts near the center of the tree cluster and begin to lower. Up close, you can make out details you couldn't before. The trees seem similar to oak trees but have leaves several times larger. Vines hang from the branches, many of which extend to other trees. As you get even closer, you realize most are not actually vines, but rather ropes which connect the trees in an intricate, scattered web. Dozens of medium-sized rafts gather in groups under the ropes and attach something between themselves. After a few minutes, they move away, revealing that they are now supporting large nets below each rope. Finally, a raft with multiple large monitors orients itself above everything and displays the symbols of the Arachi houses you noticed on the rafts earlier followed by a list of the participants from each house.`

Reviewer 3: ```suggestion
			`	The rafts near the center of the tree cluster and begin to lower. Up close, you can make out details you couldn't before. The trees seem similar to oak trees but have leaves several times larger. Vines hang from the branches, many of which extend to other trees. As you get even closer, you realize most are not actually vines, but rather ropes which connect the trees in an intricate, scattered web. Dozens of medium-sized rafts gather in groups under the ropes and attach something between themselves. After a few minutes, they move away, revealing that they are now supporting large nets below each rope. Finally, a raft with multiple large monitors orients itself above everything and displays the symbols of the Arach houses you noticed on the rafts earlier followed by a list of the participants from each house.`
```

> 			`	"Damaged, the forest would be, if built here, grand colosseums were," the Saryd explains. "Carefully cultivated, this area was, so that for the games, used it can be."`

Reviewer 3: ```suggestion
			`	"Damaged, the forest would be, if built here, grand coliseums were," the Saryd explains. "Carefully cultivated, this area was, so that for the games, used it can be."`
```
I think in English "Colosseum" is reserved for the Italian monument, while "coliseum" is used to refer to a large stadium.
Although here it's used by an alien translation device, so it's not that important.

> 			`	The participants' rafts hover near the top of the largest, central tree while they disembark, remaining at the very top of the crown as the rafts move away again. The spectator rafts then start passing close enough to the top of the tree that the crowd can see the individual contestants, and you finally spot Tumub among them. He also notices you and stretches out his legs to stand up higher, perking his head up to greet you. When the rafts have stopped moving, you are close enough to see that a symbol matching one of the Arachi houses is painted on the top of his abdomen. Glancing around, it seems most of the other contestants are adorned in the same manner.`

Reviewer 3: ```suggestion
			`	The participants' rafts hover near the top of the largest, central tree while they disembark, remaining at the very top of the crown as the rafts move away again. The spectator rafts then start passing close enough to the top of the tree that the crowd can see the individual contestants, and you finally spot Tumub among them. He also notices you and stretches out his legs to stand up higher, perking his head up to greet you. When the rafts have stopped moving, you are close enough to see that a symbol matching one of the Arach houses is painted on the top of his abdomen. Glancing around, it seems most of the other contestants are adorned in the same manner.`
```

			`	Noticing your attention, the Saryd tells you that Tumub wears the symbol for house Debdo, the Arach house that specializes in the service industries. She begins to list the other symbols and tell you which house each one represents, but a short siren cuts her off, and the contestants begin climbing down the main tree. You do your best to follow Tumub's movements, but shortly after the game begins, various groups of Arachi move rapidly across the ropes to the other trees, and you lose track of him in the confusion.`
			`	The rope web, connecting in what seems to be a random fashion, lets the Arachi spread quickly both between the tops of all the trees and up and down each one. Whenever an Arach reaches the top of a tree, they begin scouring the upper canopy, sometimes appearing to nearly burrow themselves into the leaves and branches.`

> 			`	The Arachi contestants eventually begin to find shiny, colored rings among the treetops, which they place around one of their legs, much like a bracelet, to then carry with them on to other trees. There seems to be a variety of searching methods: some work their way clockwise around the central tree, some counterclockwise, and some with no apparent pattern at all, repeatedly heading back to the center and then taking a new route.`

Reviewer 3: ```suggestion
			`	The Arach contestants eventually begin to find shiny, colored rings among the treetops, which they place around one of their legs, much like a bracelet, to then carry with them on to other trees. There seems to be a variety of searching methods: some work their way clockwise around the central tree, some counterclockwise, and some with no apparent pattern at all, repeatedly heading back to the center and then taking a new route.`
```

> 			`	As the Arachi accumulate rings, you notice they seem to slow down and begin to struggle with their balance. Two Arachi that have already been to three different treetops are both fighting to cross the same straight rope leading to a fourth when the legs of the one in front give out. As they fall down into the nets below, the shaking of the rope dislodges the other behind them. After they fall, one of the Heliarchs in your raft begins passing around some examples of the rings to the spectators so their immense weight can felt personally. When one is passed to you, it is a surprising struggle to keep it from falling; despite holding it with both hands and relying on the support of the chair, passing it off to a Kimek sitting beside you leaves you breathless. As the rings make their way back to the Heliarch, they remind the crowd on your raft that the rings are made with the same incredibly dense alloys used in the hulls of Arach ships.`

Reviewer 3: ```suggestion
			`	As the Arachi accumulate rings, you notice they seem to slow down and begin to struggle with their balance. Two Arachi that have already been to three different treetops are both fighting to cross the same straight rope leading to a fourth when the legs of the one in front give out. As they fall down into the nets below, the shaking of the rope dislodges the other behind them. After they fall, one of the Heliarchs in your raft begins passing around some examples of the rings to the spectators so their immense weight can be felt personally. When one is passed to you, it is a surprising struggle to keep it from falling; despite holding it with both hands and relying on the support of the chair, passing it off to a Kimek sitting beside you leaves you breathless. As the rings make their way back to the Heliarch, they remind the crowd on your raft that the rings are made with the same incredibly dense alloys used in the hulls of Arach ships.`
```

			`	"A test of one's endurance, this stage is," the Saryd says. "Proud of both their ships' and their own resilience, the Arachi are. To hold on the longest, the objective is." Most of the Arachi are able to stay on for roughly a minute, but soon after that, the competitors begin falling one after another. At the two minute mark, only four are left, but all fall only a few seconds later.`

> 			`	Once the results from the two contests are merged, Tumub has climbed to eighth. He and the other Arachi in the top twenty all celebrate enthusiastically, but as dark storm clouds gather on the horizon and rapidly begin approaching the cluster of trees, it isn't long before everyone is back on rafts returning to the port. When you step off the raft, you see that aside from the train that brought you here, another one is now ready to depart. As the Saryd who sat beside you leaves also disembarks, she explains that it leads to the grounds where the cooperative game will happen, but that there should still be time for you to return and see the other games.`

Reviewer 3: ```suggestion
			`	Once the results from the two contests are merged, Tumub has climbed to eighth. He and the other Arachi in the top twenty all celebrate enthusiastically, but as dark storm clouds gather on the horizon and rapidly begin approaching the cluster of trees, it isn't long before everyone is back on rafts returning to the port. When you step off the raft, you see that aside from the train that brought you here, another one is now ready to depart. As the Saryd who sat beside you also disembarks, she explains that it leads to the grounds where the cooperative game will happen, but that there should still be time for you to return and see the other games.`
```

			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping toward the wall. The Kimek and Arachi in each chariot do their best to stay on, as their Saryd teammates seem to generally press on with little regard for the chariot's stability; instead they are focused solely on reaching the wall as fast as possible.`
			`	It's a very tight race, but the Saryds begin slowing once they approach the wall, both out of exhaustion and to prevent a crash. As the Saryds arrive at the base of the cliff, they immediately lay down to rest while the Arach on their team begins to unstrap the chariot. The Kimek hurriedly begins climbing the wall with the bags they had packed earlier.`
			`	Before long, the Kimek have all reached the top of the cliff and begin opening their bags to grab a multitude of paddles, some kind of pole which looks vaguely like a combination of a shovel and pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must. My role this would have been. As skilled in anticipating the actions of my friends, I hope my former opponent is."`

> 			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems that the Arachi use these straps to carry their larger teammates up the cliff. You have your monitor focus on Pairri's friends again and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he chooses the spots that can best support the weight of both him and his Saryd friend.`

Reviewer 3: ```suggestion
			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths of fabric to secure themselves to each other. It seems that the Arachi use these straps to carry their larger teammates up the cliff. You have your monitor focus on Pairri's friends again and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he chooses the spots that can best support the weight of both him and his Saryd friend.`
```

			`	"Deemed worthy, some of them have been," Pairri says. "Name those they have chosen, the Heliarchs will." As the Heliarch captains from each ship speak, they commend the teams that have made it there, noting that their accomplishments are proof of what everyone in the Coalition can achieve when cooperating as one. Once they're done, the monitor switches to a live view of one of the athletes, a Kimek. A Saryd is next, then another Kimek, followed by an Arach. Tumub's and Ulmodi's faces are shown next, at which point Pairri repeatedly taps your arm with one of her legs.`
			`	"Selected, my friends have been!" she cheers. "Made it, they have!"`

> 			`	Most of the audience shares her excitement, and their unified cheers are a roaring chorus of congratulations for the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony and announce the end of this year's Coalition games, then invite two each of the six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs and will go to the Ring of Friendship to begin training for their new positions of honor.`

Reviewer 3: ```suggestion
			`	Most of the audience shares her excitement, and their unified cheers are a roaring chorus of congratulations for the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony and announce the end of this year's Coalition games, then invite each of the six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs and will go to the Ring of Friendship to begin training for their new positions of honor.`
```

			`	The ring platforms head back to the hangars. You and Pairri board a train that takes you back to one of the previous arenas, and from there you take another train back to the entrance to the games. During the train rides, she uses some device to call friends and family members, who congratulate her on getting so far, but even after trying several times, she can't seem to contact Ulmodi or Tumub.`
			`	"Busy with the Heliarchs, they must be." She guesses. "That there with them I am not, a shame it is."`
			choice
				`	"So are they Heliarchs now?"`
				`	"You almost won this time, so I'm sure you'll make it next time."`
					goto nexttime
			`	"Heliarchs, yes, but properly trained ones, not quite," She explains. "Still instructed in their new functions, they must be. Similar to the standard process, it is. Only, for those selected during the games, much faster said process is."`
				goto ending
			label nexttime
			`	"Thank you Captain. Even if next year watching my matches you are not, my best, I promise I will do."`
			label ending

> 			`	When you're nearing the docks, a dozen Kimek suddenly come rushing towards two of you, and Pairri tells you it's her family. She tries to introduce you to all of them, but most of them seem much more interested in telling her how proud they were to see her compete than they are in meeting you, despite your alien appearance. Only one of the younger Kimek in the family group pays you particular mind over Pairri, but in all the hustle they still don't have a chance to speak to you with her translation box. After her family calms down a bit and begin to whisk Pairri away, you say goodbye, wishing her good luck for next year's games, and then head back to your ship.`

Reviewer 3: ```suggestion
			`	When you're nearing the docks, a dozen Kimek suddenly come rushing towards the two of you, and Pairri tells you it's her family. She tries to introduce you to all of them, but most of them seem much more interested in telling her how proud they were to see her compete than they are in meeting you, despite your alien appearance. Only one of the younger Kimek in the family group pays you particular mind over Pairri, but in all the hustle they still don't have a chance to speak to you with her translation box. After her family calms down a bit and begin to whisk Pairri away, you say goodbye, wishing her good luck for next year's games, and then head back to your ship.`
```

					goto arachi
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen only kimek"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the Arachi game.)`
					goto arachi
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen only arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen saryd, kimek"
			choice
				`	(Take the train to the Arachi game.)`
					goto arachi
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen saryd, arachi"
			choice
				`	(Take the train to the Kimek game.)`
					goto kimek
				`	(Take the train to the cooperative game.)`
					goto nwp
			
			label "seen kimek, arachi"
			choice
				`	(Take the train to the Saryd game.)`
					goto saryd
				`	(Take the train to the cooperative game.)`
					goto wp
			
			label "seen all"
			choice
				`	(Take the last train.)`
					goto wp
				
			
			label saryd
			apply
				set "coalition games: saryd"

> 			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley, where the Saryd events are being held. Within the valley lies a track, stretching from a long line of starting gates then splitting as it disappears into a forest which covers most of the valley. The track apparently loops back on itself before emerging from the forest, reappearing in front of a second stadium which is located against the wall of the valley just below this one. The airspace above the valley is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just those that want a clearer view .`

Author: No, it's only one valley.
I've changed the bit after the mention of the second stadium, does that fix the issue?

> 			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley, where the Saryd events are being held. Within the valley lies a track, stretching from a long line of starting gates then splitting as it disappears into a forest which covers most of the valley. The track apparently loops back on itself before emerging from the forest, reappearing in front of a second stadium which is located against the wall of the valley just below this one. The airspace above the valley is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just those that want a clearer view .`

Reviewer 2: This is definitely an improvement, but some things are still unclear to me. Where are the starting gates? Where is the finish line? Are they inside or outside of the stadiums? Sorry if I'm being dense here.

> 			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley, where the Saryd events are being held. Within the valley lies a track, stretching from a long line of starting gates then splitting as it disappears into a forest which covers most of the valley. The track apparently loops back on itself before emerging from the forest, reappearing in front of a second stadium which is located against the wall of the valley just below this one. The airspace above the valley is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just those that want a clearer view .`

Author: Did some quick changes, do they help with the questions?

> 			`	You arrive at an elegant, crescent shaped stadium that hugs the side of a huge valley, where the Saryd events are being held. Within the valley lies a track, stretching from a long line of starting gates then splitting as it disappears into a forest which covers most of the valley. The track apparently loops back on itself before emerging from the forest, reappearing in front of a second stadium which is located against the wall of the valley just below this one. The airspace above the valley is buzzing with small drones which are transmitting live footage to dozens of cinema-sized screens placed around the stadium you're in; it seems to be for those who can't see from this distance or just those that want a clearer view .`

Reviewer 2: Definitely, thank you for putting up with me! I'll try to give one final review of this by the end of today.

			`	Finally, several dozen Saryds begin emerging from the far the other side of the forest to cross the finish line, but from what you gather it seems only the thirty highest scoring ones may proceed. The monitors switch over to a ground crew interviewing those who qualified, and you notice Ulmodi on one of the screens; he seems cheery and clearly excited by his high placement, as he seems to have finished fourth.`
			`	With the event over, the crowd disperses under the supervision of Heliarch agents, who divide the audience and guide them into rooms with many large elevators. You board the elevator your group is led to, and it begins descending down into the valley. During the descent, a Saryd Heliarch agent approaches you and begins speaking through her translation device. "Enjoyed the event, have you human?"`
			choice
				`	"It was a very entertaining event. I can see why the crowd was excited the whole time."`
					goto entertaining
				`	"I think the event would have been more interesting had I known what exactly was going on."`

> 			`	She nods in response. "Very foreign for those from outside our Coalition, I suppose our games are. Homage to the history of each of species' history, they pay, so difficult to understand at first, they can be."`

Reviewer 4: ```suggestion
			`	She nods in understanding. "Very foreign for those from outside our Coalition, I suppose our games are. Homage to the history of each of species' history, they pay, so difficult to understand at first, they can be."`
```

				goto elevatorend
			label entertaining
			`	"Glad to hear that, I am," She says. "A rare occasion it is, that to an outsider of our Coalition, our games we can show."`
			label elevatorend
			`	As the elevator finishes its descent and the doors begin to open, the Heliarch catches your attention once more. "Hope that you enjoy this next event, I do very much, though the best at it in my time I was not," she says as she begins guiding the audience along a path leading to the other stadium in the valley. Fortunately, it is much shorter than the track through the forest, so it isn't long before the escorted crowd arrives and begins filling the seats inside. In the center of this stadium, you see a rectangular court of some sort with small mounds dotted around the field and six large holes arranged evenly around the sides. Tall nets stretching to the ceiling separate the field and the seating area in a way you find to be slightly worrying.`

> 			`	Once the audience is seated, an announcer again comes on the loudspeaker, reiterating the contestants that have qualified, though in an order that seems to be different than their ranking in the last event. Sensing your confusion, a Kimek sitting next to you with a translation device explains that the winning contestants from the previous event are resting. In the meantime they have been divided into six competing teams, each assigned a different color and with total scores as close to equal as possible. Apparently each participant will still be scored individually at the end of the event based on their specific contributions, in addition to their team's overall performance. As you sit waiting for the event to start, you are reminded of how out of place your are among the alien beings, and you realize you are quite likely the first human to ever witness these games.`

Reviewer 4: ```suggestion
			`	Once the audience is seated, an announcer again comes on the loudspeaker, reiterating the contestants that have qualified, though in an order that seems to be different than their ranking in the last event. Sensing your confusion, a Kimek sitting next to you with a translation device explains that the winning contestants from the previous event are resting. In the meantime they have been divided into six competing teams, each assigned a different color and with total scores as close to equal as possible. Apparently each participant will still be scored individually at the end of the event based on their specific contributions, in addition to their team's overall performance. As you sit waiting for the event to start, you are reminded of how out of place you are among the alien beings, and you realize you are quite likely the first human to ever witness these games.`
```

> 			`	A slightly awkward half hour passes in which the uniqueness of your presence is more and more noted, before the thirty Saryds enter the arena and regain the crowd's attention. Holding what looks like fancy polo mallets, the members of each team are once more introduced to the crowd with a drone fly over, and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`

Reviewer 4: ```suggestion
			`	A half hour passes before the thirty Saryds enter the arena and regain the crowd's attention. Holding what looks like fancy polo mallets, the members of each team are once more introduced to the crowd with a drone fly over, and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle, and one member from each team stands around them.`
```
I like the previous thought about how the player is the only human to ever witness the games, but I think expanding on the awkwardness of it isn't really important to what's going on in the story.

> 			`	Once the game starts, you start to realize that it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defences. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`

Reviewer 4: ```suggestion
			`	Once the game starts, you start to realize that it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`
```

> 			`	Eventually, a sphere finds its way into a hole, which lights up red for the color of the scoring team. Much of the crowd jumps up and screams, though you can't quite tell if everyone is a particular fan of anyone on the red team or if they're cheering simply because a point was finally scored. The sphere is shot back out automatically from a tube close to the hole and is immediately back in play.`

Reviewer 4: ```suggestion
			`	Eventually, a sphere finds its way into a hole, which lights up red for the color of the scoring team. Much of the crowd jumps up and screams, though you can't quite tell whether anyone particularly favors a team or if they're cheering simply because a point was finally scored. The sphere is shot back out automatically from a tube close to the hole and is immediately back in play.`
```

> 			`	As the game continues each team scores a few times, but the blue team pulls into the lead. When they score their fifth point, the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The helpful Kimek with the translator excitedly explains to you what is happening.`

Reviewer 4: ```suggestion
			`	As the game continues, each team scores a few times, but the blue team pulls into the lead. When they score their fifth point, the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The helpful Kimek with the translator excitedly explains to you what is happening.`
```

> 			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, starting with the yellow team, the lowest scoring, and progressing up to the blue team at the end. After this event, Ulmodi has risen to being placed third, and you notice that the Saryd who caught your eye in the forest earlier is in first place. After the individual scores are announced, the top twenty contestants celebrate by running laps around the arena.`

Reviewer 4: ```suggestion
			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, starting with the yellow team, the lowest scoring, and progressing up to the blue team at the end. After this event, Ulmodi, who was on the blue team, has risen to being placed third, and you notice that the Saryd who caught your eye for perfectly clearing the forest earlier is in first place. After the individual scores are announced, the top twenty contestants celebrate by running laps around the arena.`
```
It feels strange to not mention which team Ulmodi was on, and being on blue is the simplest/shortest explanation for why his place would rise. Added some detail to the first place Saryd since there is more than one unnamed Saryd mentioned (specifically the one who's disqualified). Not necessary, but I thought these two might help for clarification.

> 			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena. At this point the crowd begins to file out, once more under Heliarch direction. You arrive back at where the train dropped you off and see that another one has arrived on the opposite side. The Heliarch agent directing your group explains that it is headed to the final arena where the cooperative game will take place, but that you are also free to travel back to the other arenas and view the rest of the other games.`

Reviewer 4: ```suggestion
			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena. At this point, the crowd begins to file out, once more under Heliarch direction. You arrive back at where the train dropped you off and see that another one has arrived on the opposite side. The Heliarch agent directing your group explains that it is headed to the final arena where the cooperative game will take place, but that you are also free to travel back to the other arenas and view the rest of the other games.`
```

				goto choice
			
			
			label kimek
			apply
				set "coalition games: kimek"

> 			`	The overly-packed train takes you to a savanna plain with a colossal cylindrical skyscraper, looking as if a stadium had been stretched vertically tenfold. Upon entering, you look up and see that, instead of a full ceiling for this level, there is only a large circular platform connected to the walls by thin beams. The underside of the platform has many individual small rings painted onto it, and a net just below that spans the entire room. Looking through the net, you can see that the same set up is repeated for the next level above, and each one above that, to the top of the tower.`

Reviewer 4: ```suggestion
			`	The overly-packed train takes you to a savanna plain with a colossal cylindrical skyscraper, looking as if a stadium had been stretched vertically tenfold. Upon entering, you look up and see that instead of a full ceiling for this level, there is only a large circular platform connected to the walls by thin beams. The underside of the platform has many individual small rings painted onto it, and a net just below that spans the entire room. Looking through the net, you can see that the same set up is repeated for the next level above, and each one above that, to the top of the tower.`
```

> 			`	You ask a nearby Heliarch agent where Pairri will be competing, and they check a computer before telling you that of the twenty floors, she's on the sixteenth. They point you towards the nearest elevator. After a ride as crowded as the train is, you rush to find a vacant seat. Though a quick glance shows there are likely enough seats on this level alone for over a hundred thousand spectators, you don't want to risk having to fight for a space given how quickly the seating area is filling up with fans all eagerly waiting for the event to begin. As you settle in, you spot many groups of Arachi and several lone Saryds, but compared to the thousands upon thousands of Kimek crowded around them, from a distance they look nearly as out of place as you feel yourself. `

Reviewer 4: ```suggestion
			`	You ask a nearby Heliarch agent where Pairri will be competing, and they check a computer before telling you that of the twenty floors, she's on the sixteenth. They point you toward the nearest elevator. After a ride as crowded as the train is, you rush to find a vacant seat. Though a quick glance shows there are likely enough seats on this level alone for over a hundred thousand spectators, you don't want to risk having to fight for a space given how quickly the seating area is filling up with fans all eagerly waiting for the event to begin. As you settle in, you spot many groups of Arachi and several lone Saryds, but compared to the thousands upon thousands of Kimek crowded around them, from a distance they look nearly as out of place as you feel yourself. `
```

			`	This process repeats several more times until finally, those that remain - less than a tenth of those that entered - are gathered in the center. As they leave the ring, without an announcement of any rankings or score, you become unsure if the game has ended.`
			`	You feel something tap the top of your head and turn to find that it is an Arach sitting behind you. "Confused, the human visitor is?" their little box translates.`
			choice
				`	"Yes. I have no idea what is going on."`
				`	"I think I somewhat understand, but an explanation would be helpful."`

> 			`	They break down the competition for you during what you learn is a brief break. "Numerous, the Kimek are, so many, many athletes competing, they have," they say. "Over a thousand, at first enter each floor. Redone, the teams are, once in half, that number is cut. The true game begins, when one hundred remain."`

Reviewer 4: ```suggestion
			`	They break down the competition for you during what you learn is a brief break. "Numerous, the Kimek are, so many, many athletes competing, they have," they say. "Over a thousand, at first enter each floor. Redone, the teams are, once in half, that number is cut. The true game begins when one hundred remain."`
```
This is a little strange. Regardless of which way you flip that last sentence, it's still structured normally - and if normal grammar rules apply (which I think they should because it's a normal sentence), that comma shouldn't be there.

> 			`	Just then, the remaining contestants reenter the ring almost as if on queue. Now, they are wearing what look like some kind of goggles, perhaps as protective gear. This time, instead of being split into two teams to fight, they are divided into pairs, and each pair moves to one of the small rings painted on the platform.`

Reviewer 4: Why wouldn't they have been wearing the goggles before?

Author: Good question.
Now they do.

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Author: I'm not sure about this, I don't think this makes the location of those monitors very clear, as "bottom of the arena" seems a bit broad.

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Reviewer 2: Forgive me if I'm misunderstanding, but the monitors are on the floor of the arena that the player is on, correct?

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Author: The seats are arranged in a ring around the "hole" in the center (not really a hole, there's no bottomless pit to fall on, but I don't know how else to describe it). From the limit of each floor, various beams extend to platforms at the center of the "hole"  on each floor. Those platforms are kept up by the beams. The fighters compete upside down, so if you're on floor one, you'd be watching the match of those on the platform that divides floors one and two, with only the top floor not having some glimpses of the floor above through the gaps. The "topside of the arena for the floor below you" means that, on the opposite side of where the fighters of floor one are competing, would be the monitors showing the match that is taking place in floor two.

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Reviewer: Each level has monitors in the center of the floor showing the stats, replays, etc for the fight on that level's ceiling.

But just saying "floor" gets confusing when the space in question is mostly "hole" and "floor" is also referring to the entire level as a whole.

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Reviewer 2: I've changed it a bit, does that help?

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Reviewer: Around the outside and down in the middle, on what would otherwise be the floor of your level, underneath the nets that catch the competitors.

The topside of the section that the level below you is fighting on the underside of.

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Reviewer 2: Ohh, okay. Thanks for the clarification!

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Reviewer: If you can word it less awkwardly, I'm all for it. ;)

Thanks btw for catching most of my little slips. I especially tend to drop smaller words when I move things around, but I never notice it unless I sit on it a while before checking again.

> 			`	As fighters lose matches by being pushed out of their circle or falling to the netting below, they don't leave like previous contestants, but instead rotate counter-clockwise to the next ring to fight again. Monitors around the outside of the arena and on the topside of the arena for the floor below you display each fighter and their changing scores as fighters rotate around the rings. Pairri is initially set to fight relatively close to where you're sitting, so you get a good view of her fights, even when she is occasionally forced to rotate away from you. While she is short even by Kimek standards, she stands her ground and often manages to beat larger opponents, and she quickly climbs the rankings even with her rare losses.`

Reviewer 2: No problem, thanks for catching my slips too :)

			`	With the event finally over, the audience begins to return to the elevators, with Heliarch agents making sure everyone proceeds down and back to the entrance in an orderly fashion. There, you see a second train in addition to the one you arrived on, and the Arach who helped explain the event to you earlier informs you that it leads to where the cooperative game will take place. The first will also take you back to where you can observe the other games.`
				goto choice
			
			
			label arachi
			apply
				set "coalition games: arachi"
			`	When the train stops, you find yourself inside a building similar to a spaceport, but the ships are more akin to a mix between a helicopter and a raft. As other visitors start boarding the flying platforms, you follow suit and find that each one is filled by dozens of adjustable seats. It seems these seats' default position is designed for Arachi, but moving a few parts makes them capable of easily accommodating Kimek and even Saryds. After fiddling with yours for a while, you manage to adjust it to a configuration which is suitable enough, inasmuch as it is more comfortable than the others.`

> 			`	Once the rafts are full, they leave their docks and depart the port. Leaving the building, you realize it was built into the side of a highland, and you're immediately soaring out over a lush rainforest. Increasingly larger trees emerge from the carpet of green below, which seems to shimmer with the movement of the various animals in the canopy. Bird-like creatures with funneled beaks, startled by the rafts' engines, appear from the large flocks and soar ahead of and away from the fleet of flying rafts before settling back into the trees. In the distance, there is a river so wide you at first believe it to be an ocean, until a Heliarch agent informs you otherwise.`

Reviewer 4: ```suggestion
			`	Once the rafts are full, they leave their docks and depart the port. Leaving the building, you realize it was built into the side of a highland, and you're immediately soaring out over a lush rainforest. Increasingly larger trees emerge from the carpet of green below, which seems to shimmer with the movement of the various animals in the canopy. Bird-like creatures with funneled beaks, startled by the rafts' engines, appear in large flocks and soar ahead of and away from the fleet of flying rafts before settling back into the trees. In the distance, there is a river so wide you at first believe it to be an ocean, until a Heliarch agent informs you otherwise.`
```

			`	"The multiple Arach houses, the symbols represent," she says. "To be patrons of athletes, a custom of members of the great houses it is."`
			`	The rafts near the center of the tree cluster and begin to lower, and up close, you can make out details you couldn't before. The trees seem similar to oak trees but have leaves several times larger. Vines hang from the branches, many of which extend to other trees. As you get even closer, you realize most are not actually vines, but rather ropes which connect the trees in an intricate, scattered web. Dozens of medium-sized rafts gather in groups under the ropes and attach something between themselves. After a few minutes, they move away, revealing that they are now supporting large nets below each rope. Finally, a raft with multiple large monitors orients itself above everything and displays the symbols of the Arachi houses you noticed on the rafts earlier followed by a list of the participants from each house.`
			`	"Damaged, the forest would be, if built here, grand colosseums were," the Saryd explains. "Carefully cultivated, this area was, so that for the games, used it can be."`

> 			`	The participants' rafts hover near the top of the largest, central tree while they disembark, remaining at the very top of the crown as the rafts move away again. The spectator rafts then start passing close enough to the top of the tree that the crowd can see the individual contestants, and you finally spot Tumub among them. He also notices you and stretches out his legs to stand up higher, perking his head up to greet you. When the rafts have stopped moving, you are close enough to see that, painted on the top of his abdomen, is a symbol matching one of the Arachi houses. Glancing around, it seems most of the other contestants are adorned in the same manner.`

Reviewer 4: ```suggestion
			`	The participants' rafts hover near the top of the largest, central tree while they disembark, remaining at the very top of the crown as the rafts move away again. The spectator rafts then start passing close enough to the top of the tree that the crowd can see the individual contestants, and you finally spot Tumub among them. He also notices you and stretches out his legs to stand up higher, perking his head up to greet you. When the rafts have stopped moving, you are close enough to see that a symbol matching one of the Arachi houses is painted on the top of his abdomen. Glancing around, it seems most of the other contestants are adorned in the same manner.`
```

			`	The rope web, connecting in what seems to be a random fashion, lets the Arachi spread quickly both between the tops of all the trees and up and down each one. Whenever an Arach reaches the top of a tree, they begin scouring the upper canopy, sometimes appearing to nearly burrow themselves into the leaves and branches.`
			`	The Arachi contestants eventually begin to find shiny, colored rings among the treetops, which they place around one of their legs, much like a bracelet, to then carry with them on to other trees. There seems to be a variety of searching methods: some work their way clockwise around the central tree, some counterclockwise, and some with no apparent pattern at all, repeatedly heading back to the center and then taking a new route.`

> 			`	As the Arachi accumulate rings, you notice they seem to slow down and begin to struggle with their balance. Two Arachi that have already been to three different treetops are both fighting to cross the same straight rope leading to a fourth when the legs of one in front give out. As they fall down into the nets below, the shaking of the rope dislodges the other behind them. After they fall, one of the Heliarchs in your raft begins passing around some examples of the rings to the spectators so their immense weight can felt personally. When one is passed to you, it is a surprising struggle to keep it from falling; despite holding it with both hands and relying on the support of the chair, passing it off to a Kimek sitting beside you leaves you breathless. As the rings make their way back to the Heliarch, they remind the crowd on your raft that the rings are made with the same incredibly dense alloys used in the hulls of Arach ships.`

Reviewer 4: ```suggestion
			`	As the Arachi accumulate rings, you notice they seem to slow down and begin to struggle with their balance. Two Arachi that have already been to three different treetops are both fighting to cross the same straight rope leading to a fourth when the legs of the one in front give out. As they fall down into the nets below, the shaking of the rope dislodges the other behind them. After they fall, one of the Heliarchs in your raft begins passing around some examples of the rings to the spectators so their immense weight can felt personally. When one is passed to you, it is a surprising struggle to keep it from falling; despite holding it with both hands and relying on the support of the chair, passing it off to a Kimek sitting beside you leaves you breathless. As the rings make their way back to the Heliarch, they remind the crowd on your raft that the rings are made with the same incredibly dense alloys used in the hulls of Arach ships.`
```

> 			`	As more and more competitors find the rings hidden in the trees, you learn that they need to retrieve a ring from each of the eight shorter trees to qualify for the next game. The order at which they approach the trees is part of their strategy; the ropes leading to sections that require vertical climbing are nearly impossible for those with many rings. Those that successfully collect all their rings without falling return to the main tree and turn them in. A scoreboard then ranks them in order and displays their completion time. There are just under one hundred who finish, and Tumub is placed thirtieth.`

Reviewer 4: ```suggestion
			`	As more and more competitors find the rings hidden in the trees, you learn that they need to retrieve a ring from each of the eight shorter trees to qualify for the next game. The order in which they approach the trees is part of their strategy; the ropes leading to sections that require vertical climbing are nearly impossible for those with many rings. Those that successfully collect all their rings without falling return to the main tree and turn them in. A scoreboard then ranks them in order and displays their completion time. There are just under one hundred who finish, and Tumub is placed thirtieth.`
```

> 			`	As more and more competitors find the rings hidden in the trees, you learn that they need to retrieve a ring from each of the eight shorter trees to qualify for the next game. The order at which they approach the trees is part of their strategy; the ropes leading to sections that require vertical climbing are nearly impossible for those with many rings. Those that successfully collect all their rings without falling return to the main tree, where they turn in their rings. A scoreboard then ranks them in order and displays their completion time. There are just under one hundred who finish, and Tumub is placed thirtieth.`

Reviewer 2: Again, reducing word repetition. 
```suggestion
			`	As more and more competitors find the rings hidden in the trees, you learn that they need to retrieve a ring from each of the eight shorter trees to qualify for the next game. The order at which they approach the trees is part of their strategy; the ropes leading to sections that require vertical climbing are nearly impossible for those with many rings. Those that successfully collect all their rings without falling return to the main tree and turn them in. A scoreboard then ranks them in order and displays their completion time. There are just under one hundred who finish, and Tumub is placed thirtieth.`
```

> 			`	The contestants are recalled back to their rafts, where they are provided with water and given time to rest. During their break, those rafts holding the nets in place below the rope web rearrange themselves to form a long, straight line, as three other rafts carrying cargo land. They are unloaded to reveal bulky, and clearly heavy, suits of armor that the remaining contestants don after their break, and those disqualified leave on them once the unloading is done. Two larger rafts with a thin, metallic pipe connecting them then approach and set themselves in position over the line of nets. The Arachi spread across the thin pipe, and the helpful Saryd explains that they must perform the difficult task of maintaining their balance on the narrow surface with their weighty armor as long as possible.` 

Reviewer 4: ```suggestion
			`	The contestants are recalled back to their rafts, where they are provided with water and given time to rest. During their break, those rafts holding the nets in place below the rope web rearrange themselves to form a long, straight line, as three other rafts carrying cargo land. They are unloaded to reveal bulky, and clearly heavy, suits of armor that the remaining contestants don after their break. Those disqualified leave on the cargo rafts once the unloading is done. Two larger rafts with a thin, metallic pipe connecting them then approach and set themselves in position over the line of nets. The Arachi spread across the thin pipe, and the helpful Saryd explains that they must perform the difficult task of maintaining their balance on the narrow surface with their weighty armor as long as possible.` 
```

> 			`	"A test of one's endurance, this stage is," the Saryd says. "Proud of both their ships' and their own resilience, the Arachi are. To hold on the longest, the objective is." Most of the Arachi are able to stay on for roughly a minute, but soon after that the competitors begin falling one after another. At the two minute mark, only four are left, but all fall only a few seconds later.`

Reviewer 4: ```suggestion
			`	"A test of one's endurance, this stage is," the Saryd says. "Proud of both their ships' and their own resilience, the Arachi are. To hold on the longest, the objective is." Most of the Arachi are able to stay on for roughly a minute, but soon after that, the competitors begin falling one after another. At the two minute mark, only four are left, but all fall only a few seconds later.`
```

			`	Once the results from the two contests are merged, Tumub has climbed to eighth. He and the other Arachi in the top twenty all celebrate enthusiastically, but as dark storm clouds gather on the horizon and rapidly begin approaching the cluster of trees, it isn't long before everyone is back on rafts returning to the port. When you step off the raft, you see that aside from the train that brought you here, another one is now ready to depart. As the Saryd who sat beside you leaves also disembarks, she explains that it leads to the grounds where the cooperative game will happen, but that there should still be time for you to return and see the other games.`
				goto choice
			
			
			label wp

> 			`	Unlike the other trains, this one blatantly mimics Heliarch warships in design, clearly meshing the three styles of Coalition ships. It brings you to a large hangar, where you're instructed to board one of hundreds of ring-shaped flying rafts available. All seats face the inside of the ring, and you take the one that looks best suited for a human. As you idly watch the mechanics here perform various preflight safety checks on the raft, you hear what you by now recognize as Kimek speech from the seat to your right.`

Author: What I suppose you could call a running joke is that all three Coalition species have very different bodies, and as such "chairs" for them would be different enough that it'd be annoying to use the chair of another species, so they have adjustable seats that can be rearranged to better fit any of the three.
Humans have different bodies than all three species, though, and as such it would be annoying to use seats in any of the three configurations, so you need to fiddle with the seats to have them comfortable. The "best suited for a human" part is saying one of the seats was left in a configuration that's easier to change into a regular human seat.
It's just another way of saying the same "the parts of these seats can be changed and you change them to be usable by humans bla bla" bit, without repeating it again.

> 			`	Unlike the other trains, this one blatantly mimics Heliarch warships in design, clearly meshing the three styles of Coalition ships. It brings you to a large hangar, where you're instructed to board one of hundreds of ring-shaped flying rafts available. All seats face the inside of the ring, and you take the one that looks best suited for a human. As you idly watch the mechanics here perform various preflight safety checks on the raft, you hear what you by now recognize as Kimek speech from the seat to your right.`

Reviewer 2: Hmm, the writing here seems to imply that there was a seat there custom-built for a human. Perhaps you could change the wording to suggest that a seat was already configured in such a way that first last doesn't have to do much adjusting to make it fit them.

> 			`	Unlike the other trains, this one blatantly mimics Heliarch warships in design, clearly meshing the three styles of Coalition ships. It brings you to a large hangar, where you're instructed to board one of hundreds of ring-shaped flying rafts available. All seats face the inside of the ring, and you take the one that looks best suited for a human. As you idly watch the mechanics here perform various preflight safety checks on the raft, you hear what you by now recognize as Kimek speech from the seat to your right.`

Author: Is what I changed it to better?

> 			`	Unlike the other trains, this one blatantly mimics Heliarch warships in design, clearly meshing the three styles of Coalition ships. It brings you to a large hangar, where you're instructed to board one of hundreds of ring-shaped flying rafts available. All seats face the inside of the ring, and you take the one that looks best suited for a human. As you idly watch the mechanics here perform various preflight safety checks on the raft, you hear what you by now recognize as Kimek speech from the seat to your right.`

Reviewer 2: Yes, much better.

			`	"Enjoyed the games, have you, Captain?" The small box translates just as you turn around, seeing that Pairri has taken her own seat besides yours.`
			choice
				`	"Yeah, though I didn't have time to stop by your own match, sorry."`
					goto notime
				`	"Wait, why are you up here with the viewers?"`
					goto why
			
			label notime
			`	She mumbles something that the box doesn't pick up, and then says, "Cost me the match, impatience did," she says. "Still, much younger than most Kimek there, I am. Glad to have gotten so far, I feel."`
				goto coop-game
			
			label prettygood
			`	"Thank you. Glad to have gotten so far, I feel. Disappointing, defeat is, but try harder next year, I shall," she says."`
				goto coop-game
			
			label why
			`	"For the Kimek victors only, the cooperative match is reserved. Team up with Tumub and Ulmodi, my opponent will."`
			
			label coop-game

> 			`	Once the mechanics are done, the ring gradually rises up in the air, and then fully takes off to leave the hangar. Out in the open, you overlook a massive track separated from the surrounding wilderness by a perfectly rectangular border of trees. Immediately under you the contestants stand near a starting line, divided into twenty groups of three. As you're brought closer to the teams, small monitors in front of you light up, and Pairri inputs a command for yours to focus on Tumub, Ulmodi, and, apparently their final teammate, the Kimek that beat Pairri in her match.`

Reviewer 4: ```suggestion
			`	Once the mechanics are done, the ring gradually rises up in the air, and then fully takes off to leave the hangar. Out in the open, you overlook a massive track separated from the surrounding wilderness by a perfectly rectangular border of trees. Immediately under you, the contestants stand near a starting line, divided into twenty groups of three. As you're brought closer to the teams, small monitors in front of you light up, and Pairri inputs a command for yours to focus on Tumub, Ulmodi, and, apparently their final teammate, the Kimek that beat Pairri in her match.`
```

			`	The monitor also allows you to switch to a camera view of the entire track. It is a long, mostly flat stretch of land, leading to a rocky cliff wall on the far side that is cut to form a perfect right angle with the ground. Just beyond the top of the cliff is a river bank with many logs laid on the ground. The river itself is not terribly wide, but the current is so strong that the various Heliarch safety boats already waiting in the waters need to constantly adjust their position in order to avoid hitting the many rock outcroppings protruding from the water. Most significantly, though, as the river approaches the tree line, it drops off in a dangerous looking waterfall to resume its course on the lower ground.`

> 			`	Pairri draws your attention back to the starting line, where Heliarch judges have the twenty teams line up. The Saryds, who seem to be working themselves up by hitting their hoofs repeatedly on the ground wait otherwise patiently. Meanwhile, their Kimek and Arach teammates pack and secure large bags and lengths of fabric in what appears to be a more conventional chariot. When the packing is complete and their teammates have boarded, the Saryds lift the yoke and strap themselves in before returning to their hoof stamping. As each team readies themselves, the Kimek and Arachi in the chariots pound on the side in time to the stamping and the teams begin to synch up, creating a uniform rhythm, like a giant heart beat.`

Reviewer 4: ```suggestion
			`	Pairri draws your attention back to the starting line, where Heliarch judges have the twenty teams line up. The Saryds seem to be working themselves up by hitting their hoofs repeatedly on the ground wait otherwise patiently. Meanwhile, their Kimek and Arach teammates pack and secure large bags and lengths of fabric in what appears to be a more conventional chariot. When the packing is complete and their teammates have boarded, the Saryds lift the yoke and strap themselves in before returning to their hoof stamping. As each team readies themselves, the Kimek and Arachi in the chariots pound on the side in time to the stamping and the teams begin to synch up, creating a uniform rhythm, like a giant heart beat.`
```

			`	"In many ways we differ, when it comes to physical form," Pairri says. "So in the cooperative match, when come together as one we do, different tasks, each member of the team is assigned. For the sake of fairness, it is so."`
			choice
				`	"What are they carrying in those bags?"`
				`	"So is pulling those chariots the task of the Saryds?"`
			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping toward the wall. The Kimek and Arachi in each chariot do their best to stay on, as their Saryd teammates seem to generally press on with little regard for the chariot's stability; instead they are focused solely on reaching the wall as fast as possible.`
			`	It's a very tight race, but the Saryds begin slowing once they approach the wall, both out of exhaustion and to prevent a crash. As the Saryds arrive at the base of the cliff, they immediately lay down to rest while the Arach on their team begins to unstrap the chariot. The Kimek hurriedly begins climbing the wall with the bags they had packed earlier.`

> 			`	Before long, the Kimek have all reached the top of the cliff and begin opening their bags to grab a multitude of paddles, some kind of pole which looks vaguely like a combination shovel and a pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must. My role this would have been. As skilled in anticipating the actions of my friends I hope my former opponent is."`

Reviewer 4: ```suggestion
			`	Before long, the Kimek have all reached the top of the cliff and begin opening their bags to grab a multitude of paddles, some kind of pole which looks vaguely like a combination of a shovel and pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must. My role this would have been. As skilled in anticipating the actions of my friends, I hope my former opponent is."`
```

> 			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems that the Arachi use these straps to carry their larger teammates up the cliff. You have your monitor focus on Pairri's friends again, and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he chooses the spots that can best support the weight of both him and his Saryd friend.`

Reviewer 4: ```suggestion
			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems that the Arachi use these straps to carry their larger teammates up the cliff. You have your monitor focus on Pairri's friends again and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he chooses the spots that can best support the weight of both him and his Saryd friend.`
```

> 			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems that the Arachi use these straps to carry their larger teammates up the cliff. You have your monitor focus on Pairri's friends again, and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he carefully chooses the best spots to anchor himself and support the weight of his Saryd friend.`

Reviewer 2: ```suggestion
			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems that the Arachi use these straps to carry their larger teammates up the cliff. You have your monitor focus on Pairri's friends again, and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he chooses the spots that can best support the weight of both him and his Saryd friend.`
```

> 			`	Once Tumub reaches the top, Ulmodi dislodges himself and removes his harness. He then lifts his Arach teammate into his arms so that he may climb onto the Saryd's back this time, while moving to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft, and Ulmodi helps his Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where he now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`

Reviewer 4: ```suggestion
			`	Once Tumub reaches the top, Ulmodi dislodges himself and removes his harness. He lifts the exhausted Arach and carries him to the raft, which is nearly finished by now. Tumub climbs into the raft, and Ulmodi helps his Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river. He now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`
```

> 			`	Once Tumub reaches the top, Ulmodi dislodges himself and removes the harness from Tumub. He then lifts his Arach teammate into his arms so that he may climb onto the Saryd's back this time, while moving to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft, and Ulmodi helps his Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where Ulmodi now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`

Reviewer 2: ```suggestion
			`	Once Tumub reaches the top, Ulmodi dislodges himself and removes his harness. He then lifts his Arach teammate into his arms so that he may climb onto the Saryd's back this time, while moving to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft, and Ulmodi helps his Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where he now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`
```

> 			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. At the same time, Ulmodi grabs the pole tool from between two of the logs. He then uses the pick edges to grab onto some rocks jutting from the river, while alternately thrusting at other rocks with the shovel portion in order to keep their small raft from dashing against them. This stabilizes the raft and helps pull them across the swift current. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft begins to make its way across the river even when Ulmodi has no convenient rock to pull against.`

Reviewer 4: ```suggestion
			`	The Kimek quickly picks up three of the paddles and works to fight the river's flow, so far without much success. At the same time, Ulmodi grabs the pole tool from between two of the logs. He then uses the pick edges to grab onto some rocks jutting from the river while alternately thrusting at other rocks with the shovel portion in order to keep their small raft from dashing against them. This stabilizes the raft and helps pull them across the swift current. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft begins to make its way across the river even when Ulmodi has no convenient rock to pull against.`
```

> 			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. At the same time, Ulmodi grabs the pole tool from between two of the logs. He then uses the pick edges to grab onto some of rocks jutting from the river, while alternately thrusting at other rocks with the shovel portion in order to keep their small raft from dashing against them. This stabilizes the raft and helps pull them across the swift current. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft begins to make its way across the river even when Ulmodi has no convenient rock to pull against.`

Reviewer 2: ```suggestion
			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. At the same time, Ulmodi grabs the pole tool from between two of the logs. He then uses the pick edges to grab onto some rocks jutting from the river, while alternately thrusting at other rocks with the shovel portion in order to keep their small raft from dashing against them. This stabilizes the raft and helps pull them across the swift current. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft begins to make its way across the river even when Ulmodi has no convenient rock to pull against.`
```

> 			`	You switch back to a more broad view of the race, and see that not everyone is having the same success crossing. Two rafts, unable to fight the stream, drift dangerously far down the river and close to the waterfall.  Others have been shattered or torn into by the rocks when the Saryd on board could not manage to push the raft away in time.  One team's raft even simply falls apart without hitting a single rock; it seems the ropes weren't properly fastened to the logs. The Heliarch safety boats on hand speed towards those in distress, plucking the competitors from the river's currents before anyone can be seriously hurt.`

Reviewer 4: ```suggestion
			`	You switch back to a more broad view of the race and see that not everyone is having the same success crossing. Two rafts, unable to fight the stream, drift dangerously far down the river and close to the waterfall.  Others have been shattered or torn into by the rocks when the Saryd on board could not manage to push the raft away in time.  One team's raft even simply falls apart without hitting a single rock; it seems the ropes weren't properly fastened to the logs. The Heliarch safety boats on hand speed towards those in distress, plucking the competitors from the river's currents before anyone can be seriously hurt.`
```

> 			`	You switch back to a more broad view of the race, and see that not everyone is having the same success crossing. Two rafts, unable to fight the stream, drift dangerously far down the river and close to the waterfall, while others have been shattered or torn into by the rocks when the Saryd on board cannot manage to push away from the rocks in time.  One team's raft even simply falls apart without hitting a single rock; it seems the ropes weren't properly fastened to the logs. The Heliarch safety boats on hand speed towards those in distress, plucking the competitors from the river's currents before anyone can be seriously hurt.`

Reviewer 2: ```suggestion
			`	You switch back to a more broad view of the race, and see that not everyone is having the same success crossing. Two rafts, unable to fight the stream, drift dangerously far down the river and close to the waterfall.  Others have been shattered or torn into by the rocks when the Saryd on board could not manage to push the raft away in time.  One team's raft even simply falls apart without hitting a single rock; it seems the ropes weren't properly fastened to the logs. The Heliarch safety boats on hand speed towards those in distress, plucking the competitors from the river's currents before anyone can be seriously hurt.`
```

			`	"Deemed worthy, some of them have been," Pairri says. "Name those they have chosen, the Heliarchs will." As the Heliarch captains from each ship speak, they commend the teams that have made it there, noting that their accomplishments are proof of what everyone in the Coalition can achieve when cooperating as one. Once they're done, the monitor switches to a live view of one of the athletes, a Kimek. A Saryd is next, then another Kimek, followed by an Arach. Tumub's and Ulmodi's faces are shown next, at which point Pairri repeatedly taps your arm with one of her legs.`

> 			`	"Selected, my friends have been!" She cheers. "Made it, they have!"`

Reviewer 4: ```suggestion
			`	"Selected, my friends have been!" she cheers. "Made it, they have!"`
```

> 			`	Most of the audience shares her excitement, and their unified cheers are a roaring chorus of congratulations for the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony and announce the end of this year's Coalition games, then invite two each of the six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs, and will go to the Ring of Friendship to begin training for their new positions of honor.`

Reviewer 4: ```suggestion
			`	Most of the audience shares her excitement, and their unified cheers are a roaring chorus of congratulations for the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony and announce the end of this year's Coalition games, then invite two each of the six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs and will go to the Ring of Friendship to begin training for their new positions of honor.`
```

> 			`	Most of the audience shares her excitement, and their unified cheers are a roaring chorus of congratulations for the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony and announce the end of this year's Coalition games, then invite two each of the six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs, and will go to the Ring of Friendship to begin training their new positions of honor.`

Reviewer 2: ```suggestion
			`	Most of the audience shares her excitement, and their unified cheers are a roaring chorus of congratulations for the six athletes. Once the crowd has settled down, the Heliarchs briefly conclude the ceremony and announce the end of this year's Coalition games, then invite two each of the six that were selected into their Punishers. Pairri explains they have been elevated to the rank of Heliarchs, and will go to the Ring of Friendship to begin training for their new positions of honor.`
```

> 			`	The ring platforms head back to the hangars. You and Pairri board a train that takes you back to one of the previous arenas, and from there you take another train back to the entrance to the games. During the train rides she uses some device to call friends and family members, who congratulate her on getting so far, but even after trying several times, she can't seem to contact Ulmodi or Tumub.`

Reviewer 4: ```suggestion
			`	The ring platforms head back to the hangars. You and Pairri board a train that takes you back to one of the previous arenas, and from there you take another train back to the entrance to the games. During the train rides, she uses some device to call friends and family members, who congratulate her on getting so far, but even after trying several times, she can't seem to contact Ulmodi or Tumub.`
```

> 			`	"Busy with the Heliarchs they must be." She guesses. "That there with them I am not, a shame it is."`

Reviewer 4: ```suggestion
			`	"Busy with the Heliarchs, they must be." She guesses. "That there with them I am not, a shame it is."`
```
